packages/ceramic-core/src/document.ts

```diff
@@ -423,8 +453,31 @@ class Document extends EventEmitter {
     this._doctype.state = state
   }
 
+  /**
+   * Loads schema for the Doctype
+   * @param ceramicApi - Ceramic API
+   * @param doctype - Doctype instance
+   */
+  static async loadSchema<T extends Doctype>(ceramicApi: CeramicApi, doctype: Doctype): Promise<T> {
+    return doctype.state?.metadata?.schema ? Document.loadSchemaById(ceramicApi, doctype.state.metadata.schema) : null
+  }
+
+  /**
+   * Loads schema by ID
+   * @param ceramicApi - Ceramic API
+   * @param schemaDocId - Schema document ID
+   */
+  static async loadSchemaById<T extends Doctype>(ceramicApi: CeramicApi, schemaDocId: string): Promise<T> {
+    if (schemaDocId) {
+      const schemaDoc = await ceramicApi.loadDocument(schemaDocId)
+      return schemaDoc.content
+    }
+    return null
+  }
+
   get content (): any {
-    return this._doctype.state.nextContent || this._doctype.state.content
+    const { next, content } = this._doctype.state
+    return next? next.content || content : content
```

Comment:
Not an issue but could be simplified as `next?.content ?? content`

Reply:
Do you mean `next?.content || content`? Or what does the `??` operator do?

Reply:
It's similar to `||` but only if the left-hand side is `null` or `undefined`.
I find it safer than `||` because it treats falsy values such as `false`, `0` and `''` as expected values on the left-hand side.

Reply:
Cool!

Reply:
nice!